src/helpers/utils/utils.ts

```diff
@@ -48,8 +48,9 @@ export function getSuccessLoggerData(guild: Guild | null, user: User, command: C
 
 export function parseBoolean(bool: string | boolean): boolean {
 	if (typeof bool === 'boolean') return bool;
-	return ['true', 't', '1', 'yes', 'y'].includes(bool.toLowerCase());
```

Comment:
Why is this removed? It could be that the provided string has uppercase letters?